### ISSUE TITLE
Integrate with Google Play Music

### ISSUE THREAD

**Issue by reya276** [artemanufrij/playmymusic#61]
How can I integrate Google Music?

**Comment by artemanufrij** [comment]
I guess an official API doesn't exist

**Comment by reya276** [comment]
Yes it does, look here: [https://unofficial-google-music-api.readthedocs.io/en/latest/](https://unofficial-google-music-api.readthedocs.io/en/latest/)

**Comment by aleksandar-stefanovic** [comment]
@reya276

The link literally says "unofficial", lol.

**Comment by reya276** [comment]
the nodeJS one could work.
